### ISSUE TITLE
Meraki Analytics CORS Blocker

### ISSUE THREAD

**Issue by jcng** [jcng/statcheck#1]
# Issue
App is currently unable to access Meraki lolstaticdata JSON endpoints (e.g. https://cdn.merakianalytics.com/riot/lol/resources/latest/en-US/champions/Lucian.json) due to lack of server-side `access-control-allow-origin: *`

I either need to figure out how to work through this with the Fetch API, or use a proxy as a workaround.

CORS Anywhere (https://github.com/Rob--W/cors-anywhere) has worked perfectly, but unfortunately the public demo requires manual opt-in. That works fine for testing the need for a proxy, but I will require a more permanent solution for development (and eventually deployment to production).

https://github.com/jcng/statcheck/blob/8e640b88558ed4d2a5cebe26d780274ee044fd2b/src/utility/DataSource.js#L43-L55

## For Development
I can:
- Continue to manually opt in to the demo for development purposes. Viability of this is contingent on how often I'd need to opt in. Not preferable, since that's not the intended purpose of the CORS Anywhere demo anywhere
- Use a chrome extension that will handle the CORS proxy for me

## For Production
- Best option will be hosting CORS Anywhere on my own server. See https://github.com/Rob--W/cors-anywhere#demo-server for more details. I'm going to need to learn more about Node.js before doing this. I was planning to do that as I move closer to deploying the app into production, so I'm going to put exploring this option on hold for now until then.
